account_invoice_margin/report/account_invoice_report.py

```diff
@@ -11,4 +11,4 @@ class AccountInvoiceReport(models.Model):
 
     def _select(self):
         select_str = super()._select()
-        return "%s, SUM(line.margin_signed) AS margin" % select_str
+        return "%s, line.margin_signed AS margin" % select_str
```

Comment:
An operator is needed as this is a measure that can be aggregated, being `SUM`, `AVG`, or other.

Reply:
Hi @pedrobaeza,
If we add the `SUM` operator, we need to add the group_by clause because in this version, the group_by was removed.
You could see that in this PR: https://github.com/odoo/odoo/commit/af2272ac56788244a165d5b064d6c867caaf7a92#diff-ff84262b2666fce15620cbaf0c216348f92e178b752e0154ce57588e9e276bea
Could we add this in our module?
In other case, It's possible to report odoo to add again the group_by clause?
I'm waitaing you response.
Thanks for your review.

Reply:
OK, I see. It seems the goal is to cache it and improve performance. But then I think we need to do something similar as in:

https://github.com/odoo/odoo/commit/af2272ac56788244a165d5b064d6c867caaf7a92#diff-2da964c74daf61be62254f9b949ed334551abfab4f9c1484d6812ccddc716f4aR149

Reply:
Hi @pedrobaeza,
I review with two runbot (13.0 and 14.0) and both are working well. I think the sum now is useless.